### PR TITLE
Connect refactor

### PR DESCRIPTION
**Pull request by namniak**

### CODE REVIEW

**Comment by namniak** [comment]
@njam3 @iranreyes PR is based off `video-player` and `howler` branches, so this review will need to wait until prev PRs are merged.

File: templates/react/src/sections/App/index.js [hunk 1]
```diff
@@ -1,94 +1,34 @@
...
-)(App);
+  mapDispatchToProps,
+  undefined,
+  {withRef: true}
+)
+
+export default class AppWrapper extends App {
+  constructor(props) {
+    super(props);
+    this.displayName = 'AppWrapper';
```
Comment (neo) [inline templates/react/src/sections/App/index.js]: Nice touch exporting with a name 👍 and I don't think it would need the `displayName` anymore.

Comment (neo) [inline templates/react/src/sections/App/index.js]: Nice touch exporting with a name 👍 and I don't think it would need the displayName anymore.

Comment (namniak) [inline templates/react/src/sections/App/index.js]: Yeah didn't seem to have any effect. Removed

**Comment by neo** [comment]
👍

**Pull request merged**